packages/next-urql/README.md

```diff
@@ -213,6 +213,38 @@ let default = (withUrqlClient(. clientOptions))(. make);
 
 The great part about writing thin bindings like this is that they are zero cost – in fact, the above bindings get totally compiled away by BuckleScript, so you get the full benefits of type safety with absolutely zero runtime cost!
 
+### Restricting Data Fetching to the Server
+
+If you want to use the Urql client in server-side-rendered mode of NextJS you have to load data in `getServerSideProps` lifecycle method. This will load your data while rendering a page on server.
+
+```
+// pages/index.tsx
+
+import { initUrqlClient, NextUrqlPageContext } from 'next-urql';
+
+export const getServerSideProps = async (ctx: NextUrqlPageContext): Promise<GetServerSidePropsResult<InitialProps>> => {
+  const graphQLClient = initUrqlClient({
+    url: 'https://graphql-pokemon.now.sh',
+    fetchOptions: {
+      headers: {
+        Authorization: `Bearer ${ctx?.req?.headers?.authorization ?? ''}`,
+      },
+    }
+  }, true) as Client;
+  const response = await graphQLClient.query(queryPokémon as unknown as string, { first: 20 }).toPromise();
+
+
+  return {
+    props: {
+      title: 'Pokédex',
+      data: response?.data?.pokemons
+    },
+  };
+};
```

Comment:
```suggestion
import { initUrqlClient, NextUrqlPageContext } from 'next-urql';

export const getServerSideProps = async (ctx) => {
  const client = initUrqlClient({
    url: /graphql',
  }, false /* set to false to disable suspense */);

  const result= await client.query(QUERY, {}).toPromise();

  return {
    props: { data: result.data }
  };
};
```

Edited for consistency and to clean up some smaller things with the copy (`fetchOptions` isn't relevant here and it was still referencing the Pokemon example